.gitignore

```diff
@@ -1,2 +1,3 @@
 /vendor
 composer.lock
+.idea/*
```

Comment:
Please move this to your global gitignore: https://help.github.com/articles/ignoring-files/